src/adapter/src/coord.rs

```diff
@@ -3037,27 +3027,10 @@ pub fn serve(
 // initializes oracles on bootstrap, once we have fully migrated to the new
 // postgres/crdb-backed oracle.
 async fn get_initial_oracle_timestamps(
```

Comment:
aha! The point where this method is used got moved above where we initialize the catalog, since this comment was written. Now we use it to determine the initial boot_ts that we pass to `Catalog::open()`. And we can't just go to the Postgres oracle (even though we could, because we know it's the only one, but ... 🤷), because we don't know yet what oracle is configured, because we only know that after we opened the catalog. 🙈

Reply:
All of this could be worked around, with more elbow grease, of course